formats/tabular-converter-plugin/README.md

```diff
@@ -0,0 +1,41 @@
+# Tabular Converter (v0.1.0)
+
+This WIPP plugin allows the tabular data conversion to `arrow` file format and vice versa. Currently this plugins handles only the vaex supported file formats.
+This plugin supports the following file formats which are convertable into `arrow` file format:
+```
+1. fcs
+2. csv
+3. hdf5
+4. fits
+5. parquet
+6. feather
+
+```
```

Comment:
Lists do not have to be in the triple ticks. Just put an empty line before the list starts and after the list ends.